edge-modules/DirectMethodSender/IDirectMethodInterface.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DirectMethodSender
+{
+    using System.Net;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public interface IDirectMethodClient
+    {
+        Task CloseClientAsync();
+
+        // Invoke direct method using a client
+        Task<HttpStatusCode> InvokeDirectMethodAsync(CancellationTokenSource cts);
+
+        // Create a client and open an instance
+        Task OpenClientAsync(IOpenClientAsyncArgs args);
```

Comment:
```suggestion
        Task OpenAsync(IOpenClientAsyncArgs args);
```

Reply:
I think you don't need IOpenClientAsyncArgs; just pass these required arguments through Wrapper's constructor as readonly properties.

Reply:
I need this since the Interface needs to have a common input argument for both ServiceClient and ModuleClient.